components/playback/controls/StimulationControls.vue

```diff
@@ -44,11 +44,17 @@
               <!-- this is here for testing the popover message -->
               <span id="start-popover-msg" style="display: none">{{ start_stim_label }}</span>
               <FontAwesomeIcon class="fontawesome_icon_class" :icon="['fa', 'play-circle']" />
+              <span v-show="is_stim_in_waiting" class="span__start-stop-spinner">
+                <FontAwesomeIcon :style="'fill: #ececed;'" :icon="['fa', 'spinner']" pulse />
+              </span>
             </div>
             <div v-if="play_state" v-b-popover.hover.bottom="stop_stim_label" title="Stop Stimulation">
               <!-- this is here for testing the popover message -->
               <span id="stop-popover-msg" style="display: none">{{ stop_stim_label }}</span>
               <FontAwesomeIcon class="fontawesome_icon_class" :icon="['fa', 'stop-circle']" />
+              <span v-show="is_stim_in_waiting" class="span__start-stop-spinner">
```

Comment:
This looks like the exact same thing that was added above, is there any reason we need to add this twice?